js/tools/gradientBrush.js

```diff
@@ -0,0 +1,50 @@
+window.tools.gradientBrush = {
```

Comment:
don't forget to put `/* global app */` at the top of this file, otherwise the linter is going to complain about ur use of the `app` variable (which is a "global variable" declared in the [index.html page](https://github.com/net-art-uchicago/paintArtware2.1/blob/9f3ac3d0f9459ba142d45a8d430e3e55e1251c4b/index.html#L35), which is a separate file)

Reply:
Done